apps/OboeTester/app/src/main/java/com/mobileer/oboetester/BaseAutoGlitchActivity.java

```diff
@@ -257,6 +257,7 @@ protected TestResult testConfigurations() throws InterruptedException {
             // Set output size to a level that will avoid glitches.
             AudioStreamBase outStream = mAudioOutTester.getCurrentAudioStream();
             int sizeFrames = outStream.getBufferCapacityInFrames() / 2;
+            sizeFrames = Math.max(sizeFrames, 2 * outStream.getFramesPerBurst());
```

Comment:
If 2 * outStream.getFramesPerBurst() is greater than outStream.getBufferCapacityInFrames(), we may overflow.

Can we add an additional Math.min(sizeFrames, outStream.getBufferCapacityInFrames())?

Reply:
setBufferSizeInFrames() is clipped in Oboe and AAudio against the capacity. So this is safe.
It returns the actual size after clipping and quantizing to multiples of the burst size.

Also we try to ensure the capacity is at least two bursts.